### PR TITLE
Newsart 542 Storybook

### PR DESCRIPTION
**Pull request by jamesbhobbs**
JIRA: https://jira.dev.bbc.co.uk/browse/NEWSART-542

Added storybook (the alpha due to a type error that is a breaking change), added story because I was dissatisfied by the testing of storybook installation being successful without at least one story showing.

* [x] JIRA ticket added
* [x] Pull request URL added to JIRA ticket
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* ~~[ ] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)~~
* ~~[ ] Tests added for new features~~
* [x] Tests pass - `npm test`
* ~~[ ] E2E tests pass - `npm run test:e2e` (requires server running see [README]~~(https://github.com/bbc/simorgh#tests) for details)

* Tests to be added in: https://github.com/bbc/simorgh/issues/44
* [ ] Test engineer approval

### CODE REVIEW

File: src/app/components/story.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
```
Comment (jtart) [inline src/app/components/story.jsx]: The convention for storybook filenames is `[component].stories.jsx`

Also, why is eslint disabled?

File: package.json [hunk 2]
```diff
@@ -50,6 +51,8 @@
     "speculate": "^1.7.4"
   },
   "devDependencies": {
+    "@storybook/react": "^4.0.0-alpha.8",
+    "babel-core": "^6.26.3",
```
Comment (jtart) [inline package.json:55]: Why is this being added?

**Review by jtart — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (jamesbhobbs) [reply package.json:55]: https://storybook.js.org/basics/guide-react/#add-react-react-dom-and-babel-core

**Review by jtart — APPROVED** [review]

File: src/app/components/Article.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
```
Comment (pjlee11) [inline src/app/components/Article.stories.jsx:2]: Why is the ignore linting required?

Comment (jamesbhobbs) [reply src/app/components/Article.stories.jsx:2]: @pjlee11 because it thinks it should be a dep not a dev-dep and it's wrong.

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Pull request merged**